x-pack/plugins/apm/public/components/app/transaction_details/distribution/index.tsx

```diff
@@ -105,115 +106,120 @@ export function TransactionDistribution({
     useTransactionDistributionChartData();
 
   return (
-    <div data-test-subj="apmTransactionDistributionTabContent">
-      <EuiFlexGroup style={{ minHeight: MIN_TAB_TITLE_HEIGHT }}>
-        <EuiFlexItem style={{ flexDirection: 'row', alignItems: 'center' }}>
-          <EuiTitle size="xs">
-            <h5 data-test-subj="apmTransactionDistributionChartTitle">
-              {i18n.translate(
-                'xpack.apm.transactionDetails.distribution.panelTitle',
-                {
-                  defaultMessage: 'Latency distribution',
-                }
-              )}
-            </h5>
-          </EuiTitle>
-        </EuiFlexItem>
-        {hasData && !selection && (
-          <EuiFlexItem>
-            <EuiFlexGroup justifyContent="flexEnd" gutterSize="xs">
-              <EuiFlexItem
-                grow={false}
-                style={{ flexDirection: 'row', alignItems: 'center' }}
-              >
-                <EuiIcon type="iInCircle" title={emptySelectionText} size="s" />
-              </EuiFlexItem>
-              <EuiFlexItem
-                grow={false}
-                style={{ flexDirection: 'row', alignItems: 'center' }}
-              >
-                <EuiText size="xs">{emptySelectionText}</EuiText>
-              </EuiFlexItem>
-            </EuiFlexGroup>
-          </EuiFlexItem>
-        )}
-        {hasData && selection && (
-          <EuiFlexItem grow={false}>
-            <EuiBadge
-              iconType="cross"
-              iconSide="left"
-              onClick={onTrackedClearSelection}
-              onClickAriaLabel={clearSelectionAriaLabel}
-              iconOnClick={onTrackedClearSelection}
-              iconOnClickAriaLabel={clearSelectionAriaLabel}
-              data-test-sub="apmTransactionDetailsDistributionClearSelectionBadge"
-            >
-              {i18n.translate(
-                'xpack.apm.transactionDetails.distribution.selectionText',
-                {
-                  defaultMessage: `Selection: {formattedSelection}`,
-                  values: {
-                    formattedSelection: getFormattedSelection(selection),
-                  },
-                }
-              )}
-            </EuiBadge>
+    <HeightRetainer>
```

Comment:
Github made a mess here, but I just wrapped this component with `HeightRetainer`

Reply:
Hiding whitespace changes makes this much more readable: https://github.com/elastic/kibana/pull/119233/files?diff=unified&w=1